src/controllers/commands/CommandController.cpp

```diff
@@ -2936,6 +2937,137 @@ void CommandController::initialize(Settings &, Paths &paths)
             return runWhisperCommand(words, channel);
         });
     }
+
+    auto formatVIPListError = [](HelixListVIPsError error,
+                                 const QString &message) -> QString {
+        using Error = HelixListVIPsError;
+
+        QString errorMessage = QString("Failed to list VIPs - ");
+
+        switch (error)
+        {
+            case Error::Forwarded: {
+                errorMessage += message;
+            }
+            break;
+
+            case Error::Ratelimited: {
+                errorMessage += "You are being ratelimited by Twitch. Try "
+                                "again in a few seconds.";
+            }
+            break;
+
+            case Error::UserMissingScope: {
+                // TODO(pajlada): Phrase MISSING_REQUIRED_SCOPE
+                errorMessage += "Missing required scope. "
+                                "Re-login with your "
+                                "account and try again.";
+            }
+            break;
+
+            case Error::UserNotAuthorized: {
+                // TODO(pajlada): Phrase MISSING_PERMISSION
+                errorMessage += "You don't have permission to "
+                                "perform that action.";
+            }
+            break;
+
+            case Error::UserNotBroadcaster: {
+                errorMessage += "Due to Twitch restrictions, "
+                                "this command can only be used by the broadcaster. "
+                                "To see the list of VIPs you must use the "
+                                "Twitch website.";
```

Comment:
```suggestion
                errorMessage +=
                    "Due to Twitch restrictions, "
                    "this command can only be used by the broadcaster. "
                    "To see the list of VIPs you must use the "
                    "Twitch website.";
```
if all the checks weren't stuck the formatter would be screaming about this

Reply:
I saw that went over 80 chars and did nothing about it. I will now do something about it.

Reply:
I have done something about it in 74dba0a911634e4c193444d1884dbc0b7999a59f